Assets/ComethSDK/Scripts/Tools/SaveLoadPersistentData.cs

```diff
@@ -39,11 +39,18 @@ public static EncryptionData Load(string folder, string file)
 			// get the data path of this save data
 			var dataPath = GetFilePath(folder, file);
 
-			// if the file path or name does not exist, return the default SO
+			// if the file Directory does not exist, return the default SO
 			if (!Directory.Exists(Path.GetDirectoryName(dataPath)))
 			{
-				Debug.LogWarning("File or path does not exist! " + dataPath);
-				throw new Exception("File or path does not exist! " + dataPath);
+				Debug.LogWarning("Directory does not exist! " + dataPath);
+				throw new Exception("Directory does not exist! " + dataPath);
+			}
+			
+			// if the file does not exist, return the default SO
```

Comment:
Comment is wrong. You are not returning default, you are throwing an error.
Anyway I think the code is clear enought, no nned for comment.